README.md

```diff
@@ -1,17 +1,11 @@
-# free-for.dev
+# Free For Developers
 
 Developers and Open Source authors now have a massive amount of services offering free tiers, but it can be hard to find them all to make informed decisions.
 
 This is a list of software (SaaS, PaaS, IaaS, etc.) and other offerings that have free tiers for developers.
 
-The scope of this particular list is limited to things that infrastructure developers (System Administrator, DevOps Practitioners, etc.) are likely to find useful. We love all the free services out there, but it would be good to keep it on topic. It's a bit of a grey line at times so this is a bit opinionated; do not be offended if I do not accept your contribution.
-
-This list is the result of Pull Requests, reviews, ideas and work done by 900+ people. You too can help by sending [Pull Requests](https://github.com/ripienaar/free-for-dev) to add more services or by remove ones whose offerings have changed or been retired.
```

Comment:
all of this has to stay in the list, the links, the description, everything.

After years of maintaining this there's on constant: PEOPLE ARE LAZY.

No-one is going to read about.md, removing this will make more work not less

Reply:
Ok @ripienaar merging about.md into readme.

Reply:
@ripienaar done.